backend/app/views/spree/admin/products/new.html.erb

```diff
@@ -6,60 +6,73 @@
 
     <legend align="center"><%= Spree.t(:new_product) %></legend>
 
-    <%= f.field_container :name do %>
-      <%= f.label :name, class: 'required' %><br />
-      <%= f.text_field :name, :class => 'fullwidth title', :required => true %>
-      <%= f.error_message_on :name %>
-    <% end %>
+    <div class="row">
+      <div class="col-lg-8 col-xs-12">
+        <div class="row">
+          <div class="col-xs-12">
+            <div class="row">
+              <% unless @product.has_variants? %>
+                <div data-hook="new_product_sku" class="col-md-6 col-xs-12">
```

Comment:
We probably want to find a way not to need to do this. We should only use row where it is needed